API documentation.md

```diff
@@ -63,6 +172,225 @@ Endpoint description
 This API uses the following error codes:
 
 - `400 Bad Request`: The request was malformed or missing required parameters.
-- `401 Unauthorized`: The API key provided was invalid or missing.
-- `404 Not Found`: The requested resource was not found.
+- `401 Unauthorized`: You are unauthorized.
+- `404 Not Found`: The requested document was not found.
+- `500 Internal Server Error`: An unexpected error occurred on the server.
+
+## DELETE `/documents/{id}`
+
+Delete an existing document by its unique identifier and all its links.
+
+### Response body
+
```

Comment:
I would remove this and rely on NO_CONTENT status code for signaling

Reply:
Updated Response bodies of DELETE requests